staging/src/k8s.io/client-go/plugin/pkg/client/auth/azure/README.md

```diff
@@ -45,4 +46,4 @@ This plugin provides an integration with Azure Active Directory device flow. If
    To sign in, use a web browser to open the page https://aka.ms/devicelogin and enter the code DEC7D48GA to authenticate.
```

Comment:
Actually, this step in the setup in my case could only be done on Windows. However, that was because of some iOS specific settings in the ADFS (with which the AAD was connected) that were somehow also breaking Mac browsers. I'm not sure if I should mention that here, as it might just be a very specific edge case.